tests/http/batching.yml

```diff
@@ -72,3 +72,20 @@ assert:
               - column: 1
                 line: 1
             message: " --> 1:1\n  |\n1 | FOO\n  | ^---\n  |\n  = expected executable_definition"
+  # Sending a bad request
+  - request:
+      method: GET
+      url: http://localhost:8080/api/posts?$=title,user.name
+    response:
+      status: 200
+      body:
+        data: {}
+  # Bad formatting with spaces
+  - request:
+      method: GET
+      url: http://localhost:8080/api/user ?id=1&$=id
```

Comment:
Well there's no harm in checking the spaces...

Reply:
well, if you put space in any place in url for current tab, it won't work =)

It is not obvious why should we have any custom logic that in practice shouldn't be used and in case of using spaces they should be properly encoded for http